Watchman.Tests/Dynamo/DynamoAlarmTests.cs

```diff
@@ -209,6 +209,7 @@ public async Task AlarmCreatedWithCorrectProperties()
                     && alarm.Properties["ComparisonOperator"].Value<string>() == "GreaterThanOrEqualToThreshold"
                     && alarm.Properties["Statistic"].Value<string>() == "Sum"
                     && alarm.Properties["Namespace"].Value<string>() == AwsNamespace.DynamoDb
+                    && alarm.Properties["TreatMissingData"].Value<string>() == "notBreaching"
```

Comment:
could we add the opposite test to maybe one of the other service types? just so we prove in one place that it isn't using "notBreaching" everywhere

Reply:
Yep, good idea.